core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -592,7 +599,9 @@ private[spark] class TaskSetManager(
    * failures (this is because the method picks on unscheduled task, and then iterates through each
    * executor until it finds one that the task hasn't failed on already).
    */
-  private[scheduler] def abortIfCompletelyBlacklisted(executors: Iterable[String]): Unit = {
+  private[scheduler] def abortIfCompletelyBlacklisted(
+      executorsByHost: HashMap[String, HashSet[String]],
+      blacklist: BlacklistTracker): Unit = {
```

Comment:
looks like you don't need to pass this in anymore, since it's now part of the class?